lib/store/src/StoriesListStore.ts

```diff
@@ -14,29 +14,38 @@ export class StoriesListStore {
     this.fetchStoriesList = fetchStoriesList;
   }
 
-  async initialize() {
-    return this.cacheStoriesList();
-  }
+  initialize(sync: false): Promise<void>;
+
+  initialize(sync: true): void;
 
-  initializeSync() {
-    return this.cacheStoriesListSync();
+  initialize(sync: boolean) {
```

Comment:
ESLint doesn't let us put these lines next to each other 🤷

Reply:
For consistency with StoryStore, perhaps this should accept an object with `sync` property as parameter, rather than a `sync` parameter directly.